config/src/lib.rs

```diff
@@ -197,6 +198,9 @@ pub struct Config {
     pub model_checker: Option<ModelCheckerSettings>,
     /// verbosity to use
     pub verbosity: u8,
+    /// Information for decoding custom precompile calls
+    #[serde(default)]
+    pub precompiles: Vec<(Address, Function)>,
```

Comment:
this is reasonable,

I'd like to see some tests/examples for this in toml format